pipeline/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+pandas
+datetime
```

Comment:
Datetime doesn't need to be installed - it's actually a built-in python package.